src/components/NavBar/EditorDesignName.tsx

```diff
@@ -0,0 +1,48 @@
+import {
+  Flex,
+  IconButton,
+  useEditableControls,
+  Editable,
+  EditablePreview,
+  Input,
+  EditableInput,
+} from "@chakra-ui/react";
+import { useState } from "react";
+
+import { BiStar, BiPencil } from "react-icons/bi";
+
+const DesignName = () => {
+  const [value, setValue] = useState("Court Canva 1");
+  const EditableControls = () => {
+    const { isEditing, getEditButtonProps } = useEditableControls();
+    return isEditing ? null : (
+      <IconButton
+        aria-label="Edit"
+        icon={<BiPencil />}
+        variant="navbarIconBtn"
+        {...getEditButtonProps()}
+      />
+    );
+  };
+
+  return (
+    <Flex justifyContent="center" alignItems="center" fontSize="xl">
+      <IconButton aria-label="Star" icon={<BiStar />} variant="navbarIconBtn" />
+      <Editable
+        display="inline-block"
+        color="white"
+        fontSize="xl"
+        textAlign="center"
+        isPreviewFocusable={false}
+        onChange={(value) => setValue(value)}
+        value={value}
+      >
+        <EditablePreview p="8px" />
```

Comment:
Before:
<img width="1059" alt="image" src="https://user-images.githubusercontent.com/61776449/170976011-c32a1c58-3d99-4569-9457-e955bc748c2d.png">
After:
<img width="963" alt="image" src="https://user-images.githubusercontent.com/61776449/170976093-ec3291b8-5705-43f8-9b00-5d53658bf0ee.png">
The height here will change after clicking the modify button, and I feel that the navbar suddenly becomes smaller. This experience is not very good. I think the height should be fixed.

After:

Reply:
Update and solve the problem. It's because the padding I add .